css/nav.css

```diff
@@ -0,0 +1,36 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-weight: 600;
+  font-size: 2rem;
+  margin: 0 auto;
+  width: 80%;
+}
+.nav-pills {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  padding: 1rem 0rem;
+}
+.nav-pills li {
+  margin: 0 1rem;
+}
+.nav-pills a {
+  color: rgb(90, 90, 90);
```

Comment:
add formatting for 'a' element in the reset css file, so that it will get applied to all 'a' tags.